include/seqan3/alignment/pairwise/execution/execution_handler_parallel.hpp

```diff
@@ -117,7 +117,7 @@ class execution_handler_parallel
         // Asynchronously pushes the alignment job as a task to the queue.
         task_type task = [=, indexed_sequence_pairs = std::move(indexed_sequence_pairs)] ()
         {
-            delegate(algorithm(std::move(indexed_sequence_pairs)));
+            algorithm(std::move(indexed_sequence_pairs), delegate);
```

Comment:
now it's called delegate and not callback? Are they synonyms or is there a difference?

Reply:
It was called like this before and yes they have the same meaning.